### ISSUE TITLE
proxy env not work

### ISSUE THREAD

**Issue by tkkcc** [neonmoe/minreq#95]
```toml
[dependencies]
minreq = { version = "2.8.0", features=  ["proxy"] }
```
```rust
fn main() {
    let proxy = minreq::Proxy::new("http://localhost:1080").unwrap();
    let response = minreq::post("http://example.com")
        .with_proxy(proxy)
        .send()
        .unwrap();
}
```
above panic for "failed to lookup address information: Name or service not known" 
below panic too with env `http_proxy=http://127.0.0.1:1080`
```rust
fn main() {
    let response = minreq::post("http://example.com")
        .send()
        .unwrap();
}
```

**Comment by neonmoe** [comment]
Apparently we haven't handled the "http://" part in proxies at all. Your example should work if you change the proxy to `127.0.0.1:1080`. I'll publish a new version that handles the http:// properly soon.

**Comment by neonmoe** [comment]
Fixed in version 2.8.1. Thanks for the issue report!
